### ISSUE TITLE
Accessibility: search button contains no text for readers

### ISSUE THREAD

**Issue by K-T-O** [CJSCommonPlatform/govuk_single_page_pdk#102]
Pa11y files an error about the following issue:

- WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name

Error: This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.

This relates to the search button functionality of the "gov-search" component.

**Comment by fabiotisci** [comment]
@scarlz will look into this ;)
